.bmad/bmb/workflows/edit-module/README.md

```
@@ -0,0 +1,187 @@
+
+This workflow integrates with:
+
+- **edit-agent** - For editing individual agents
+- **edit-workflow** - For editing individual workflows
+- **create-agent** - For adding new agents
+- **create-workflow** - For adding new workflows
+
+When you select options to manage agents or workflows, the appropriate specialized workflow is invoked automatically.
+
+## Module Structure
+
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)